agent.py

```diff
@@ -255,6 +292,7 @@ def main():
     if command in START_COMMANDS:
         log.info('Agent version %s' % get_version())
 
+    # MAYBE: Add reload command.. problem is that when called this way, agent.collector is None
```

Comment:
I think it's a no, we were trying to move away from these ad hoc commands, basically separate the management of the processes from the processes themselves.